PoGo.Necrobot.Window/App.xaml.cs

```diff
@@ -86,9 +88,22 @@ private void Application_Startup(object sender, StartupEventArgs e)
 
             Task.Run(() =>
             {
-                NecroBot.CLI.Program.RunBotWithParameters(OnBotStartedEventHandler, false, new string[] { });
+                NecroBot.CLI.Program.RunBotWithParameters(OnBotStartedEventHandler, new string[] { });
             });
 
+            if (Settings.Default.ConsoleToggled == true)
```

Comment:
Remove This! This will cause the App to Open Console on Launch each time. I had fixed this before so idk why it returns here